packages/layout/package.json

```diff
@@ -34,9 +34,11 @@
     "@react-pdf/stylesheet": "^4.2.4",
     "@react-pdf/textkit": "^4.4.1",
     "@react-pdf/types": "^2.4.1",
+    "canvas": "^2.11.2",
     "cross-fetch": "^3.1.5",
     "emoji-regex": "^10.3.0",
     "queue": "^6.0.1",
+    "skia-canvas": "https://github.com/samizdatco/skia-canvas/releases/download/v1.0.1/skia-canvas-v1.0.1-linux-x64-glibc.tar.gz",
```

Comment:
Why isn't this on npm?

Reply:
Before it available on npm but latest version is still not available on npm and it's recommendation to use this version from github once it available in npm we will update to get from npm.